requirements.txt

```diff
@@ -1 +1 @@
-pyhOn==0.17.3
+pyhOn==0.17.5
```

Comment:
In `hon` folder there is the `manifest.json` file and I think it should be changed there as well (https://developers.home-assistant.io/docs/creating_integration_manifest/#requirements). `requirements.txt` is more for pip to quickly install all deps needed for development.